person.rb

```diff
@@ -7,9 +7,10 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
The same here for rentals 🔥💪🏻💯💯💯
- [ ] I think it would be great if you make retals `attr_accessor` as well since also we need to set and get the rental not just get it since `attr_reader` it is just get the rental so kindly update it ⭐🚀✅📌🔥